Git LFS is required for this repo; add some documentation and a build… (#5)
hall/little-buddy: CONTRIBUTING.md

> **NOTE**: until they're no longer experimental, you'll need to [enable flakes](https://nixos.wiki/wiki/Flakes#Enable_flakes)

> ## Enable LFS
> 
> **IMPORTANT:*** You must enable git LFS; otherwise, the `.a` files included in this repo will be stubs rather than the necessary binaries.
> 
> ## build
> 

Reviewer: I wouldn't add a separate section just for this; especially since it should only pertain to the `build` section below.

```suggestion
## build

> **WARN**: git LFS must be enabled to resolve various `.a` binary files

```
